spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestWriteAborts.java

```diff
@@ -74,20 +77,16 @@ public static Object[][] parameters() {
     };
   }
 
-  @Rule public TemporaryFolder temp = new TemporaryFolder();
+  @TempDir private Path temp;
```

Comment:
`CatalogTestBase` already has `temp`, so this shouldn't be required

Reply:
Thanks for the quick review. Will remove this in the next commit.